### ISSUE TITLE
how to choose points automatically when apply the inverse projection transform?

### ISSUE THREAD

**Issue by JempChou** [vigneshr97/Udacity-Self-Driving-Car-Engineer#1]
you said "I chose to hardcode the source and destination points" when you get the bird"s eye view.But it's not convenient , right?

**Comment by vigneshr97** [comment]
Yes, it is not so convenient to give the source points. To detect them automatically using computer vision methods, edge and corner detections could be applied after using appropriate color threshold. Machine learning methods like clustering and neural network can also be used by separating the lane lines. For a 3 dimensional version, I thought the following paper would be helpful https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3355419/
